### ISSUE TITLE
learning rate

### ISSUE THREAD

**Issue by mumingerlai** [ChrisChen1023/HINT#3]
Thanks for your code. I see that the experimental settings in your paper are like this: "The learning rate is initially set to 1e-4 and is halved at the 75% milestone of the training progress." but I don't find the halving of the training progress setting in your code. Could you tell me where it is?

**Comment by ChrisChen1023** [comment]
> Thanks for your code. I see that the experimental settings in your paper are like this: "The learning rate is initially set to 1e-4 and is halved at the 75% milestone of the training progress." but I don't find the halving of the training progress setting in your code. Could you tell me where it is?

Hi there,

Thanks for your interest. For the learning rate, you could use 'torch.optim.lr_scheduler.MultiStepLR()' to set the milestones, or you could just save the ckeckpoint and resume the training with the smaller learning rate. In our experiment we use the latter plan. Hope this could help.

**Comment by mumingerlai** [comment]
Thank you very much for your answer. I understand.
